service-api/docker-compose.env

```
@@ -1,5 +1,7 @@
 MOCK_NINO_API=true
 MOCK_DRIVINGLICENSE_API=true
+MOCK_YOTI=true
+LOCAL_YOTI_BASE_URL=
```

I don't think this needs a `LOCAL_` prefix? Other similar ones don't (e.g. `DRIVINGLICENSE_BASE_URL`)
```suggestion
YOTI_BASE_URL=
```